app/workers/carry_over_unsubmitted_applications_worker.rb

```diff
@@ -0,0 +1,24 @@
+class CarryOverUnsubmittedApplicationsWorker
+  include Sidekiq::Worker
+
+  def perform
+    unsubmitted_applications_from_earlier_cycle.each do |application_form|
+      CarryOverApplication.new(application_form).call
+    end
+  end
+
+private
+
+  def unsubmitted_applications_from_earlier_cycle
+    # TODO: Remove hard-coded year
```

Comment:
```suggestion
```

Reply:
Removed as part of change following Michael's comment